app-misc/elasticsearch/elasticsearch-6.1.1.ebuild

```diff
@@ -42,16 +42,18 @@ src_install() {
 	chmod +x "${ED}"/usr/share/${PN}/bin/* || die
```

Comment:
please do the same sort of edits here too:

1. default first in `src_prepare`
2. doins -r config/.

etc